### PR TITLE
Remove backward ops for cuDNN convolution

### PR DESCRIPTION
**Pull request by jbschlosser**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69902
* __->__ #69901

Differential Revision: [D33093796](https://our.internmc.facebook.com/intern/diff/D33093796)

### CODE REVIEW

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093796).

**Review by albanD — APPROVED** [review]
Op removal 🎉

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093796).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093796).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093796).

File: aten/src/ATen/native/native_functions.yaml [hunk 1]
```diff
@@ -1430,18 +1430,6 @@
   dispatch:
     CUDA: cudnn_convolution
 
-- func: cudnn_convolution_backward_input(int[] self_size, Tensor grad_output, Tensor weight, int[] padding, int[] stride, int[] dilation, int groups, bool benchmark, bool deterministic, bool allow_tf32) -> Tensor
-  dispatch:
-    CUDA: cudnn_convolution_backward_input
```
[the remark this answers is not in the document]
Comment (albanD) [reply aten/src/ATen/native/native_functions.yaml:1435]: Following the cleanup, you should see only the `convolution` and `convolution_backward` functions that correspond to these calls.

[the remark this answers is not in the document]
Comment (robieta) [reply aten/src/ATen/native/native_functions.yaml:1435]: It's something of a historical anomaly that these calls go through the dispatcher. As a general rule the dispatcher can be thought of as managing the flow through public symbols (conv, add, etc.), and then whatever calls the implementations make are opaque. This is why, for instance, you don't expect to see calls to MKL or a particular BLAS library on the trace. We could put manual RECORD_FUNCTION macros around some of the library boundaries, effectively trading off a small amount of dispatch overhead for more visibility; however that would be a larger discussion. This change just brings conv_backward back in line with the rest of PyTorch.

[the remark this answers is not in the document]
Comment (qmpzzpmq) [reply aten/src/ATen/native/native_functions.yaml:1435]: @albanD I found the `convolution_backward` is removed as weill. is there any another method?

**Pull request closed without merging**